src/wwwroot/controllers/MainCtrl.js

```diff
@@ -81,6 +107,13 @@
         isFreeTrialEndToday : true
       }
     }
+
+    function freeTrialNotificated () {
+      auth.addFreeTrialNotificationToStorage(todayDate);
+      $rootScope.getFreeTrialNotificationFromStorage = auth.getFreeTrialNotificationFromStorage();
```

Comment:
Is it necessary to call `auth.getFreeTrialNotificationFromStorage()?, what do you think about to directly store `todayDate` in the local variable?